include/pagmo/utils/constrained.hpp

```diff
@@ -53,8 +53,8 @@ namespace detail
 
 /// Tests equality constraints against some tolerance vector. Returns number of constraints satisfied and the L2 norm of
 /// the violation
-template <typename It>
-std::pair<vector_double::size_type, double> test_eq_constraints(It ceq_first, It ceq_last, It tol_first)
```

Comment:
I checked and there is no crash. Simply the returned norm of the violation will be NaN and that particular constraint will be considered as violated.